Scripts/script-CommonServer.yml

```diff
@@ -116,7 +116,7 @@ script: |-
           t.forEach(function(entry){
               var vals = [];
               headers.forEach(function(h){
-                  vals.push(!!entry[h] ? stringEscapeMD(formatCell(entry[h])) : ' ');
+                  vals.push((!!entry[h] || entry[h]==0) ? stringEscapeMD(formatCell(entry[h])) : ' ');
```

Comment:
If I'm not mistaken, default JS is to try and convert to int using this syntax and if the entry is a string it might throw. Please verify.